mod-linked-data/src/main/resources/citation/mod-linked-data/features/lccn-pattern-validation/samples/valid-instance-request.json

```diff
@@ -14,7 +14,7 @@
         {
           "http://library.link/identifier/LCCN":{
             "http://bibfra.me/vocab/lite/name":[
-              "  1234567890"
```

Comment:
This lccn is same as the lccn used in create-bib-api test. Hence, search results return count '2' instead of '1' in create-bib-api  tests and hence cause failure.